manticore/platforms/evm.py

```diff
@@ -2381,3 +2382,27 @@ def compare_buffers(a, b):
             
         self.current._push(value)
         self.current.pc += self.current.instruction.size
+
+    def generate_workspace_files(self):
+        def solve_data(data):
+            ret = ''
+            try:
```

Comment:
Why not state.solve_one   ?

Reply:
there is no state accessible from the platform

Reply:
Ah.
So why not 
`solver.get_value(self._constraints, data)`
?

Reply:
ah, there is no reason, i wasn't paying attention :( just quickly copying `SLinux.generate_workspace_file`